sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.1.0-preview.7 (Unreleased)
 
+### Breaking Changes
+- `JsonPatchDocument` type moved to `Azure.Core` package.
```

Comment:
Maybe we don't care enough about the Expirmental package, but you could also use the TypeForwardToAttribute like the BCL has done. In that case, this isn't breaking, per se, but still worth mentioning.

Reply:
Yeah, we don't care enough about this in Experimental. And in the case of this type we've changed the namespace which AFAIK doesn't allow us to use TFTA.